src/beatmap.c

```diff
@@ -185,6 +185,28 @@ static void parse_one_hit(char *line, struct oshu_hit **hit)
 }
 
 /**
+ * Parse #oshu_hit::combo and #oshu_hit::combo_seq of a single #oshu_hit
+ */
+static void parse_hit_combo(struct oshu_hit *previous, struct oshu_hit *hit)
+{
+	int new_combo = hit->type & OSHU_HIT_NEW_COMBO;
```

Comment:
This is used only once, so I think injecting it into line 196 simplies the code. One less variable to keep track of is a probably a good thing.